dev-resources/alignments/simple.json

```diff
@@ -1,6 +1,7 @@
-{"mbox::mailto:bob@example.org":
- {"https://example.org/activity/a": 0.5,
-  "https://example.org/activity/c": -0.2},
- "mbox::mailto:alice@example.org":
- {"https://example.org/activity/c": 0.7,
-  "https://example.org/activity/d": -0.02}}
+[{"id": "mbox::mailto:bob@example.org",
+  "type": "Agent",
+  "alignments":
+  [{"component": "https://example.org/activity/a",
+    "weight": 0.5},
+   {"component": "https://example.org/activity/c",
+    "weight": -0.2}]}]
```

Comment:
Where's Alice? is the intention to change output here? This might be what's breaking yer tests